src/pages/cli/graphql/override.mdx

```diff
@@ -76,6 +76,45 @@ export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
   resources.models["Todo"].modelDatasource.dynamoDbConfig['deltaSyncConfig']['baseTableTtl'] = '3600'
 }
 ```
+### Example - Configure DynamoDBBillingMode
+
+Set the DynamoDB billing mode for the API. One of "PROVISIONED" or "PAY_PER_REQUEST".
+
+```ts
+import { AmplifyApiGraphQlResourceStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+  resources.models['Post'].modelDDBTable.billingMode = 'PAY_PER_REQUEST'
+}
+```
+### Example - Configure DynamoDBModelTableProvisionedThroughput
+Override the default [ProvisionedThroughput](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html#cfn-dynamodb-table-provisionedthroughput) provisioned for each `@model` table
+
+Only valid if the "DynamoDBBillingMode" is set to "PROVISIONED"
```

Comment:
Once we change the table's billing mode to "PROVISIONED", I believe we need to provide provisionedThroughput for all the GSIs on the table too. It would be nice to verify and add that info.